flight/Modules/ManualControl/inc/control.h

```diff
@@ -29,14 +30,18 @@
 #ifndef CONTROL_H
 #define CONTROL_H
 
-// The enum from here is used to determine the flight mode
-#include "flightstatus.h"
-
-enum control_events {
-	CONTROL_EVENTS_NONE,
-	CONTROL_EVENTS_ARM,
-	CONTROL_EVENTS_ARMING,
-	CONTROL_EVENTS_DISARM
+enum control_status {
+	STATUS_DISCONNECTED,	///< RX is disconnected
+	STATUS_ERROR,           ///< Something is fundamentally wrong
+	STATUS_SAFETYTIMEOUT,	///< Disconnected timeout has occurred
+		// Lower layer (transmitter control) responsible for timer.
+	STATUS_DISARM,		///< User requested disarm, or low throt timeout
+		// Lower layer (transmitter control) responsible for timers.
+	STATUS_NONE,		///< Things are "normal"
```

Comment:
Yah, that's a good point-- will fix.  It comes from the old 'NONE' arming events.  But now that we're level, not edge triggered...